prime-router/src/main/kotlin/fhirengine/translation/hl7/schema/ConfigSchemaReader.kt

```diff
@@ -134,7 +157,8 @@ object ConfigSchemaReader : Logging {
                 readOneYamlSchema(file.inputStream(), schemaClass)
             }
             "classpath" -> {
-                val input = javaClass.classLoader.getResourceAsStream(schemaUri.path.substring(1))
+                val schemaYml = if (schemaUri.path.endsWith(".yml")) schemaUri.path else schemaUri.path + ".yml"
```

Comment:
Shouldn't we need this change in all the cases here?  Also, can we just put `.yml` in the schema and avoid this extra branch?

Reply:
yea, here I put some heuristic tweaking, the code could be tolerant or restrict, can mandate all schema reference to yml file be explicitly ending with .yml

but even with that required in authoring schema yaml files, do we want to be tolerant (guess a suffix) or throw error e.g. can not read schema: classpath: /.../xyz ??

Reply:
My current thinking is that we should insist on the extension, but happy to discuss further.  Just seems clearer, since `classpath: /.../xyz` isn't actually a file and I was thinking we should make folks be explicit